Project.toml

```diff
@@ -13,6 +13,7 @@ CubicSplines = "9c784101-8907-5a6d-9be6-98f00873c89b"
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
+Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
```

Comment:
do we need Glob?

Reply:
Nope, sorry, that was me playing around and it stayed in there